### DISCUSSION TITLE
'dotnet workload install maui' installs old version

### DISCUSSION THREAD

**Discussion by jsiemensLatitudegeo** [dotnet/maui#19507] [Q&A]
I have .net sdk 8.0.100 installed, which is the latest, but when I run 'dotnet workload install maui', I get 8.0.0-rc.2.9373. The latest version of maui is 8.0.3, and this is what Visual Studio will install. How can I get maui 8.0.3 workload from command line? Trying to use VS code for development, but it seems like it's still necessary to use VS to install the correct version of MAUI. But VS is deprecated on Mac, so trying to use VS Code and command line tooling.

```
> sudo dotnet workload install maui          

Welcome to .NET 8.0!
---------------------
SDK Version: 8.0.100

Successfully installed workload(s) maui.

> dotnet workload list

Installed Workload Id      Manifest Version                  Installation Source
--------------------------------------------------------------------------------
maui                       8.0.0-rc.2.9373/8.0.100-rc.2      SDK 8.0.100 
```

**Comment by nll** [comment]
Does `dotnet workload update` work ?

**Reply by softeip** [reply]
This does not work. I have the same problem. I was using .net 7 and needed to switch to .net 8. I installed SDK 8.0.200 and cannot install non-rc maui version. All I get is 8.0.0-rc.1.9171/8.0.100-rc.1
Any other ways to fix?

**Reply by davidortinau** [reply]
It could be the advertising manifest is old in your cache, though this usually only an issue when you have downgraded. Try `dotnet workload uninstall maui` and then in your user directory open the `.dotnet` folder and remove the contents of the `sdk-advertising` folder. 

Confirm your NuGet sources. You ideally only want to install from public NuGet. If necessary clean that up.

```
dotnet nuget list source
```

Then install again and see if you can get 8.0.6 which is latest stable (8.0.7 is only a nuget package today and not part of the workload yet).

**Reply by softeip** [reply]
@davidortinau thank you! Now I have 8.0.6

**Reply by davidortinau** [reply]
I now see @softeip mentioned 8.0.200 for which we just posted a known issue and are working to ship a new installer that addresses that. https://github.com/dotnet/maui/issues/20600

@jsiemensLatitudegeo does any of this address your issue?
